src/Microsoft.ML.Transforms/RffTransform.cs

```diff
@@ -213,126 +209,89 @@ private void InitializeFourierCoefficients(int rowSize, int colSize)
             + "shift-invariant kernel.";
 
         public const string LoaderSignature = "RffTransform";
+
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "RFF FUNC",
-                verWrittenCur: 0x00010001, // Initial
+                //verWrittenCur: 0x00010001, // Initial
```

Comment:
Is the commented out code left here intentionally to keep track of what changed? If not, I would delete.

Reply:
This is kinda our pattern how we handle situations with bumping version.
looks weird, but you have some hopes what previous version got started from 10001 and not something else.

---
In reply to: [222860021](https://github.com/dotnet/machinelearning/pull/1122#discussion_r222860021) [](ancestors = 222860021)